doc/man3/EC_GROUP_new.pod

```diff
@@ -59,86 +58,114 @@ objects
 
-Within the library there are two forms of elliptic curve that are of interest. The first form is those defined over the
-prime field Fp. The elements of Fp are the integers 0 to p-1, where p is a prime number. This gives us a revised
+Within the library there are two forms of elliptic curve that are of interest.
+The first form is those defined over the prime field Fp. The elements of Fp are
+the integers 0 to p-1, where p is a prime number. This gives us a revised
 elliptic curve equation as follows:
 
 y^2 mod p = x^3 +ax + b mod p
 
-The second form is those defined over a binary field F2^m where the elements of the field are integers of length at
-most m bits. For this form the elliptic curve equation is modified to:
+The second form is those defined over a binary field F2^m where the elements of
+the field are integers of length at most m bits. For this form the elliptic
+curve equation is modified to:
 
 y^2 + xy = x^3 + ax^2 + b (where b != 0)
 
-Operations in a binary field are performed relative to an B<irreducible polynomial>. All such curves with OpenSSL
-use a trinomial or a pentanomial for this parameter.
+Operations in a binary field are performed relative to an
+B<irreducible polynomial>. All such curves with OpenSSL use a trinomial or a
+pentanomial for this parameter.
 
-A new curve can be constructed by calling EC_GROUP_new_ex, using the implementation provided by B<meth> (see
-L<EC_GFp_simple_method(3)>) and associated with the library context B<ctx>
-(see L<OPENSSL_CTX(3)>).
-The B<ctx> parameter may be NULL in which case the default library context is used.
+A new curve can be constructed by calling EC_GROUP_new_ex(), using the
+implementation provided by B<meth> (see L<EC_GFp_simple_method(3)>) and
+associated with the library context B<ctx> (see L<OPENSSL_CTX(3)>).
+The B<ctx> parameter may be NULL in which case the default library context is
+used.
 It is then necessary to call EC_GROUP_set_curve() to set the curve parameters.
 EC_GROUP_new_from_ecparameters() will create a group from the
 specified B<params> and
-EC_GROUP_new_from_ecpkparameters() will create a group from the specific PK B<params>.
+EC_GROUP_new_from_ecpkparameters() will create a group from the specific PK
+B<params>.
 
-EC_GROUP_new is the same as EC_GROUP_new_ex() except that the library context
+EC_GROUP_new() is the same as EC_GROUP_new_ex() except that the library context
 used is always the default library context.
 
-EC_GROUP_set_curve() sets the curve parameters B<p>, B<a> and B<b>. For a curve over Fp B<b>
-is the prime for the field. For a curve over F2^m B<p> represents the irreducible polynomial - each bit
-represents a term in the polynomial. Therefore there will either be three or five bits set dependent on whether
-the polynomial is a trinomial or a pentanomial.
+EC_GROUP_set_curve() sets the curve parameters B<p>, B<a> and B<b>. For a curve
+over Fp B<b> is the prime for the field. For a curve over F2^m B<p> represents
```

Comment:
b is the prime?? Isn't p the prime!!? (I see this is in the original text)

Reply:
Good catch! Fixing it now!